client/src/components/Admin_userlist/admin_userlist.js

```diff
@@ -95,50 +81,43 @@ export default function StickyHeadTable() {
     const user = useSelector((state) => state.user);
     const dispatch = useDispatch();
 
-        if (auth.authenticate) {
-        return <Redirect to={"/"} />;
-    }
 
-    if (user.loading) {
-        return <Progress mb='4' w='25%'><Progress.Bar striped animated min='0'max='100' mx='auto' now='50'>Loading....</Progress.Bar></Progress>;
```

Comment:
Why did you remove this line?

Reply:
Oh sorry that is unintentional.
Will correct it.